WordPress/src/main/java/org/wordpress/android/ui/main/MeViewModel.kt

```diff
@@ -135,10 +128,12 @@ class MeViewModel
             is ApiFetchedResult -> if (this.isError()) {
                 RecommendAppUiState(this.error!!)
             } else {
-                RecommendAppUiState(
+                val openSheetData = RecommendAppUiState(
                         link = this.link,
                         message = this.message
                 )
+                analyticsUtilsWrapper.trackRecommendAppEngaged(ME)
+                openSheetData
```

Comment:
Another way I've been handling this kind of thing is by using `.also`:

```suggestion
                RecommendAppUiState(
                        link = this.link,
                        message = this.message
                ).also {
                    analyticsUtilsWrapper.trackRecommendAppEngaged(ME)
                }
```

I think readability improves a bit, but it's totally a matter of preference, so feel free to ignore this.

Reply:
totally agree...and I continue to forget about `also` :)